libraries/Microsoft.Bot.Builder/ActivityHandler.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Yes there is a mix, but I was consistent in using raw Task where it was a direct result (possibly delegated) and only the in the case where it actually had some control flow following the nested asynchronous call using async await.

Actually I'm not clear which ones you think we should change? Can you just update this with exactly the one.

Reply:
OK resolving this. The model, consistently applied across all our code, is if there is control flow after the async call then use await otherwise don't both. Another way of saying this is don't use ContinueWith.